### ISSUE TITLE
Diagram not visible in high contrast theme (for accessibility)

### ISSUE THREAD

**Issue by rosensilva** [wso2/mi-vscode#506]
### Current Limitation

This is a requirement for improved accessibility and usability of technology for individuals with disabilities (VPAT). The following diagram is not visible in high contrast mode.

<img width="1840" alt="Screenshot 2024-11-08 at 18 32 27" src="https://github.com/user-attachments/assets/3629c0ed-addc-44a0-9654-8944e534b83a">

<img width="1840" alt="Screenshot 2024-11-08 at 18 33 00" src="https://github.com/user-attachments/assets/a4a67c54-ad3a-4a0f-8881-c8d8d18b26c4">

### Suggested Improvement

The diagram lines should be visible in high contrast mode for individuals with visibility challenges

**Comment by tharindulak** [comment]
Closing this since the fix is available in v2.1.5
